### PR TITLE
Mention properties-maven-plugin and commons-lang in upgrade guide.

### PR DESCRIPTION
**Pull request by julianladisch**

### CODE REVIEW

**Review by adamdickmeiss — APPROVED** [review]

File: doc/upgrading.md [hunk 1]
```diff
@@ -119,6 +119,9 @@ on the artifact with generated interfaces.
 
 ```
 
+Use `properties-maven-plugin` to set system properties needed by
+`domain-models-maven-plugin` if any.
```
Comment (hjiebsco) [inline doc/upgrading.md:123]: Add an example section will be helpful

File: doc/upgrading.md [hunk 2]
```diff
@@ -134,6 +137,9 @@ Replace any joda class usage by a java.time class usage. RMB no longer ships wit
 joda-time that is deprecated because the replacement java.time has been in JDK core
 since Java 8.
 
+Add commons-lang:commons-lang:2.6 dependency to pom.xml if commons-lang is used.
```
Comment (hjiebsco) [inline doc/upgrading.md:140]: Has to be that `commons-lang:2.6` specific version?

Comment (julianladisch) [reply doc/upgrading.md:140]: Yes, that's the latest 2.* version: https://mvnrepository.com/artifact/commons-lang/commons-lang

Comment (hjiebsco) [reply doc/upgrading.md:140]: > Yes, that's the latest 2.* version: https://mvnrepository.com/artifact/commons-lang/commons-lang

Would https://mvnrepository.com/artifact/org.apache.commons/commons-lang3 not work?

Comment (julianladisch) [reply doc/upgrading.md:140]: https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/index.html resides in org.apache.commons.lang package.
https://commons.apache.org/proper/commons-lang/javadocs/api-release/index.html resides in org.apache.commons.lang3 package.

File: doc/upgrading.md (hunk 1 above)
Comment (julianladisch) [reply doc/upgrading.md]: Link to example added.

**Review by hjiebsco — APPROVED** [review]

**Review by adamdickmeiss — APPROVED** [review]

**Pull request merged**